archunit/src/main/java/com/tngtech/archunit/core/importer/DomainBuilders.java

```diff
@@ -443,6 +444,11 @@ JavaConstructor construct(JavaConstructorBuilder builder, ImportedClasses import
         private Set<JavaModifier> modifiers = new HashSet<>();
 
         JavaClassBuilder() {
+            this(false);
+        }
+
+        JavaClassBuilder(boolean stub) {
```

Comment:
Yeah, this was kind of conscious, because it's only used internally in a limited scope, that's why I took the shortcut to just expose the boolean parameter :wink: But I guess a tiny factory method also won't hurt and convey the meaning way better :+1: